### ISSUE TITLE
Bug in emplot2d

### ISSUE THREAD

**Issue by shadowwalkersb** [cryoem/eman2#199]
### Data file to plot:
```
1 2 3
2 3 4
5 4 3
3 3 4
```

### Steps to reproduce:
1. Run e2display.py
1. Select file to plot
1. Click `Plot 2D`
1. Click one of the data points

```
$ Traceback (most recent call last):
  File "/minicondas/envs/eman-deps-11-gui/lib/python2.7/site-packages/eman2_gui/emplot2d.py", line 861, in mousePressEvent
    self.update_selected((event.x(),event.y()),lc)
  File "/miniconda/envs/eman-deps-11-gui/lib/python2.7/site-packages/eman2_gui/emplot2d.py", line 780, in update_selected
    if r[srt[i]]<3 : self.selected.append(srt[i])
IndexError: index 4 is out of bounds for axis 0 with size 4
```

**Comment by shadowwalkersb** [comment]
Does this mean that it will work only with more than 4 points?

**Comment by sludtke42** [comment]
If you look at the comment, the selection mode displays up to 5 points, and naively assumes there are at least 5 points in the data set. This would be a pretty unusual situation in any actual use-case, but is a bug. I will patch it.


--------------------------------------------------------------------------------------
Steven Ludtke, Ph.D. <sludtke@bcm.edu <mailto:sludtke@bcm.edu>>                      Baylor College of Medicine 
Charles C. Bell Jr., Professor of Structural Biology
Dept. of Biochemistry and Molecular Biology                      (www.bcm.edu/biochem <http://www.bcm.edu/biochem>)
Academic Director, CryoEM Core                                        (cryoem.bcm.edu <http://cryoem.bcm.edu/>)
Co-Director CIBR Center                                    (www.bcm.edu/research/cibr <http://www.bcm.edu/research/cibr>)
Co-Director National Center For Macromolecular Imaging                  (ncmi.bcm.edu <http://ncmi.bcm.edu/>)

**Comment by shadowwalkersb** [comment]
Fixed by 8b91e43f104c63ad712fca4fe673174b70bbddc3.
